client/my-sites/customer-home/components/celebrate-launch-modal.jsx

```diff
@@ -19,31 +20,17 @@ function CelebrateLaunchModal( { setModalIsOpen, site, allDomains } ) {
 	const isPaidPlan = ! site?.plan?.is_free;
 	const isBilledMonthly = site?.plan?.product_slug?.includes( 'monthly' );
 	const transformedDomains = allDomains.map( createSiteDomainObject );
-	const [ isInitiallyLoaded, setIsInitiallyLoaded ] = useState( false );
 	const [ clipboardCopied, setClipboardCopied ] = useState( false );
 	const clipboardButtonEl = useRef( null );
 	const hasCustomDomain = Boolean(
 		transformedDomains.find( ( domain ) => ! domain.isWPCOMDomain )
 	);
+	const sitePlanSlug = useSelector( ( state ) => getSitePlanSlug( state, site?.ID ) );
 	const purchases = useSelector( ( state ) => getSitePurchases( state, site?.ID ) );
```

Comment:
With the current changes, the `purchases` aren't getting populated. Looking at other places in the codebase, looks like we could also rely on `useQuerySitePurchases();`.

Reply:
In fact, it's getting populated if the owner of the site is the one logged in. In a way, it makes sense, as the owner is the one putting a custom domain to the site. On the other hand, that makes the launch popup to have the wrong message. I'll take a second thought about this.

Reply:
> In fact, it's getting populated if the owner of the site is the one logged in. In a way, it makes sense, as the owner is the one putting a custom domain to the site. On the other hand, that makes the launch popup to have the wrong message. I'll take a second thought about this.

Hmm. Shouldn't any admin be able to add a custom domain though? 🤔

Reply:
I have been testing and it turns out admins (not owners) can attach domains into the site too, so it makes sense to populate the purchases for the admins too. I'll put back my original code that fetched the purchases for everyone.

Reply:
The logic to show the form only when the purchases are fetched is going back too.